src/components/ModalPayment/ModalAmount/ModalAmount.tsx

```diff
@@ -66,28 +65,36 @@ export const Modal = (props: Props) => {
         return t('purchase.voucher', { seller: sellerName });
       case ModalPaymentTypes.modalPages.light_up_chinatown:
         return t('purchase.donation_to', { seller: sellerName });
+      case ModalPaymentTypes.modalPages.mega_gam:
+        return t(
+          'purchase.mega_gam',
+          sellerName ? { seller: sellerName } : { seller: 'Seller is blank' }
```

Comment:
we aren't going to have a seller for mega gam. i'm not sure where we're holding the "December Campaign" text. @justintmckibben we should probably add a name field to the campaign model for this OR we could use the project name

Reply:
got it. can wait until we decide that and then add it in